### PR TITLE
Adding global tags element 

### LINKED ISSUE #223: Operation tags in global tags element
**Problem description**

Spectral linting rule [operation-tag-defined](https://docs.stoplight.io/docs/spectral/4dec24461f3af-open-api-rules#operation-tag-defined)
produces warnings for qod-api.yaml like this:

```
  Message :   Operation tags must be defined in global tags.
  Rule    :   operation-tag-defined
  Path    :   paths./sessions.post.tags.0
  Line    :   88
```

**Expected behavior**
Tags should be defined in a top-level `tags` element

**Alternative solutions**
1. Removing tags from operation objects in qod-api.yaml 

2. Exclusion of this rule from future ruleset for CAMARA APIs as `tags` element is **not required** by OAS

https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.2.md#fixed-fields

> A list of tags used by the specification with additional metadata. The order of the tags can be used to reflect on their order by the parsing tools. Not all tags that are used by the [Operation Object](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.2.md#operationObject) must be declared. The tags that are not declared MAY be organized randomly or based on the tools' logic. Each tag name in the list MUST be unique.

**Additional context**
 Spectral default OAS ruleset includes also:

[openapi-tags-alphabetical](https://docs.stoplight.io/docs/spectral/4dec24461f3af-open-api-rules#openapi-tags-alphabetical)

> 
> OpenAPI object should have alphabetical tags. This will be sorted by the name property.
> Recommended: No

[tag-description](https://docs.stoplight.io/docs/spectral/4dec24461f3af-open-api-rules#tag-description)

> Tags alone are not very descriptive. Give folks a bit more information to work with.
> ```
> 
> tags:
>   - name: "Aardvark"
>     description: Funny-nosed pig-head raccoon.
>   - name: "Badger"
>     description: Angry short-legged omnivores.
> ```
> 
> If your tags are business objects then you can use the term to explain them a bit. An 'Account' could be a user account, company information, bank account, potential sales lead, or anything. What is clear to the folks writing the document is probably not as clear to others.
> Recommended: No

### PR DESCRIPTION
**Pull request by rartych**
#### What type of PR is this?

* correction

#### What this PR does / why we need it:

Operation tags should be defined in a top-level `tags` element.

#### Which issue(s) this PR fixes:

Fixes #223

#### Special notes for reviewers:

Description field is rendered by Swagger this way:
QoD_tags_in_Swagger

#### Changelog input

```
Global tags element added

```

### CODE REVIEW

File: code/API_definitions/qod-api.yaml [hunk 1]
```diff
@@ -81,6 +81,11 @@ servers:
       basePath:
         default: qod/v0
         description: Base path for the QoD API
+tags:
+  - name: QoS sessions
+    description: Manage QoS sessions
+  - name: QoS Profiles
+    description: Manage QoS Profiles
```
Comment (hdamker) [inline code/API_definitions/qod-api.yaml:88]:
```suggestion
    description: Manage QoS Profiles
```

**Review by hdamker — COMMENTED** [review]
One small correction necessary to make linting happy ;-)

**Review by jlurien — COMMENTED** [review]
We should use same format for both tags, either both sessions and profiles in lowecase or uppercase. Tags per operation have to be adjusted accordingly

**Comment by hdamker** [comment]
@jlurien

> We should use same format for both tags, either both sessions and profiles in lowecase or uppercase. Tags per operation have to be adjusted accordingly

I looked over the documentation parts, and there we have also the same difference, it's more or less consequently "QoS session" and "QoS Profile". But I propose to not extend the scope of this PR to a cleanup of the documentation parts, and do it here only for the tags. 

I would tend to lowercase "QoS session" and "QoS profile", but that's just my view. Any other views?

**Comment by eric-murray** [comment]
> I would tend to lowercase "QoS session" and "QoS profile", but that's just my view. Any other views?

My preference is Title Case, but this is a style point and really one for Commonalities and the design guidelines., whose [example tags](https://github.com/camaraproject/Commonalities/blob/main/documentation/images/guidelines-fig-15.png) are unfortunately ambiguous on this point.

**Comment by rartych** [comment]
I do no think we need exact guidelines for tags style/format, as tags are helper properties to clearly describe and render OAS and are useful with large API specifications.
I have aligned to the current style of tags, with the reasoning that while sessions are dynamic and created on request, QoD Profiles were initially predefined and static.
But I agree that having just "QoS session" and "QoS profile" is good solution.
Should the tags and their descriptions be updated in this PR?
Or together with separate issue for unification of the style in all the documentation?

**Comment by eric-murray** [comment]
If we want a common "look and feel" between CAMARA APIs, then we need an agreement in Commonalities on tag format style.

**Review by RandyLevensalor — APPROVED** [review]
LGTM  Thanks for adding this.

**Review by eric-murray — APPROVED** [review]

**Review by hdamker — APPROVED** [review]

**Comment by hdamker** [comment]
Good to go. Thanks @rartych !

**Pull request merged**